hwcomposer/hwcomposer_backend_v20.cpp

```diff
@@ -195,25 +203,14 @@ void HWC2Window::present(HWComposerNativeWindowBuffer *buffer)
 
```

Comment:
Not part of the changes, just struck my eye that there is this "presentAndGetReleaseFences" error check that has no chances of getting triggered. Some leftover code from past?

Reply:
That would have been checking a block I removed in the first commit. Updated to remove the check too.